lib/Parser/RegexPattern.cpp

```diff
@@ -41,7 +41,7 @@ namespace UnifiedRegex
             Assert(
                 !GetScriptContext()->GetDynamicRegexMap()->TryGetValue(
                     RegexKey(source.GetBuffer(), source.GetLength(), GetFlags()),
-                    &p) ||
+                    &p) || ( source.GetLength() == 0 ) ||
```

Comment:
We have a builtin regexp stored in the map that are not cleaned up by default. It's ok for it to be there.